packages/services/api/src/modules/token/resolvers.generated.ts

```diff
@@ -0,0 +1,20 @@
+/* This file was automatically generated. DO NOT UPDATE MANUALLY. */
```

Comment:
I think maybe this one can be removed from source control?

Reply:
Yes! This is done in https://github.com/kamilkisiela/graphql-hive/pull/4937/commits/051a7ffe8d145f9b1785096981132c67c38a6957